ssh_tarpit/utils.py

```diff
@@ -12,6 +12,14 @@ def setup_logger(name, verbosity):
                                            '%(name)s: %(message)s',
                                            '%Y-%m-%d %H:%M:%S'))
     logger.addHandler(handler)
+    if logfile:
+      fh = logging.FileHandler(logfile)
+      fh.setLevel(level=verbosity)
+      fh.setFormatter(logging.Formatter('%(asctime)s '
+                                        '%(levelname)-8s '
+                                        '%(name)s: %(message)s',
+                                        '%Y-%m-%d %H:%M:%S'))
+      logger.addHandler(fh)
```

Comment:
Please use space indentation which is multiple of four spaces for consistency. I.e. add 2 more spaces.